pawn/Resources/Callbacks/OnPlayer/OnPlayerCommandText.pwn

```diff
@@ -873,10 +877,18 @@ public OnPlayerCommandText(playerid, cmdtext[]) {
                 player, playerLevel);
 
             // If the user was temp'd, show admins who temp'd the player.
-            if (tempLevel[player] == 1 || tempLevel[player] == 2) {
-                if (Player(playerid)->isAdministrator())
+            {     
```

Comment:
What happened here? There's brackets without an if statement or whatso ever.

You might've made a mistake?